src/comments.js

```diff
@@ -0,0 +1,129 @@
+          try {
+            const response = await fetch(baseUrl + appsEndPoint + id + commentsEndPoint, {
+              method: 'POST',
+              headers: {
+                'Content-Type': 'application/json',
+              },
+              body: JSON.stringify(comment),
+            });
+            const data = await response.text();
+            return data;
+          } catch (error) {
+            return error;
+          }
+        }
+      });
+    }
+
+      static getComments = async (itemId) => {
+        try {
+          const res = await fetch(baseUrl + appsEndPoint + id + CommentEndPt + itemId, {
+            method: 'GET',
+            headers: {
+              'Content-Type': 'application/json',
+            },
+          });
+          const data = await res.text();
+          return data;
+        } catch (error) {
+          return error;
+        }
+      }
+
+      static displayPopUp = (e) => {
+        if (e.target.classList.contains('comment-btn')) {
+          const id = e.target.getAttribute('id');
+          const detail = SHOWS.allShows.find((item) => Number(item.id) === Number(e.target.id));
+          let detailResult = '';
+          COMMENTS.getComments(id).then((comments) => {
+            if (comments.length) {
+              JSON.parse(comments).forEach((comment) => {
+                detailResult += `<li>${comment.creation_date} ${comment.username}: ${comment.comment}</li>`;
+              });
+            }
+
+            const popup = `<div class="popup-card">
+              <button id="desktop-popup-close">
+                  <img src= ${closeIcon} alt="close">
+             </button>
+              <div class="popup-img-wrap">
+                  <img src=${detail.image.medium} alt="popup">
+                  <button id="popup-close">
+                      <img id="close-img" src=${closeIcon} alt="close">
+                 </button>
+              </div>
+              <div class="title-wrap">
+                  <h2>${detail.name}</h2>
+              </div>
+              <div class="more-details-wrap">
+                  <div class="more-details">
+                      <p class="comment"><span class="origin">premiered</span>: ${detail.premiered}</p>
+                      <p class="comment"><span class="origin">status</span>: ${detail.status}</p>
+                  </div>
+                  <div class="more-details align-right">
+                      <p class="comment"><span class="origin">Country</span>: ${detail.network.country.name}</p>
+                      <p class="comment"><span class="origin">Language</span>: ${detail.language}</p>
+                  </div>
+              </div>
+              <div class="comment-wrap">
+                  <p>comments <span id="comments-count">${(JSON.parse(comments) && JSON.parse(comments).length) ? JSON.parse(
+              comments,
+            ).length : 0}</span></p>
+              </div>
+              <ul class="commentors">${detailResult}</ul>
```

Comment:
- Kindly, we need need to refresh the page to view the newly added comment, so please update it to have better user interaction :+1: 
![Screenshot from 2023-03-17 23-01-29](https://user-images.githubusercontent.com/52010789/226021336-43b05fc9-8f85-4c15-bbe7-91b4df74e757.png)